utils/guardrails/guardrails.yaml

```diff
@@ -0,0 +1,74 @@
+S1: 
```

Comment:
What is the source of this yaml file?
A reference would be helpful.

Reply:
added:
# Default guardrails taken from https://llama.meta.com/docs/model-cards-and-prompt-formats/meta-llama-guard-2/